superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -454,13 +454,30 @@ const forceSSLField = ({
       }}
     />
     <span css={toggleStyle}>SSL</span>
-    <InfoTooltip
-      tooltip={t('SSL Mode "require" will be used.')}
-      placement="right"
-      viewBox="0 -5 24 24"
-    />
+
+    {db?.database_name === 'PostgreSQL' && (
+      <InfoTooltip
+        tooltip={t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "require" will be used.')}
+        placement="right"
+        viewBox="0 -5 24 24"
+      /> )}
+
+    {db?.database_name === 'MySQL' && (
+      <InfoTooltip
+        tooltip={t('Requires a root certificate authority (public, local, or self-signed).')}
+        placement="right"
+        viewBox="0 -5 24 24"
+      /> )}
+
+    {db?.database_name === 'Amazon Redshift' && (
+      <InfoTooltip
+        tooltip={t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "verify-ca" will be used.')}
+        placement="right"
+        viewBox="0 -5 24 24"
+      /> )}
+
   </div>
-);
+  )
```

Comment:
This parenthesis should have no tabbed space before it and a semicolon after it. Run the Prettify extension in VScode to catch these little things before GitHub CI does